aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -39,6 +39,19 @@ TORCH_META_FUNC(addmm)(const Tensor& self, const Tensor& mat1, const Tensor& mat
   "The input tensor must be a matrix with size ", mat1.sizes()[0], "x", mat2.sizes()[1], ", but got a ", result.dim(),
   "-D tensor with size ", result.sizes()[0], "x", result.sizes()[1]);
 }
+
+TORCH_META_FUNC(mm)(const Tensor & self, const Tensor & mat2) {
+  TORCH_CHECK(self.dim() == 2, "self must be a matrix");
+  TORCH_CHECK(mat2.dim() == 2, "mat2 must be a matrix");
+
+  auto names = at::namedinference::compute_matmul_outnames(self, mat2);
+  set_output(0, IntArrayRef({self.sizes()[0], mat2.sizes()[1]}), {}, self.options(), names);
```

Comment:
nit: don't need to explicitly cast the initializer list to IntArrayRef, there's an implicit conversion